Styles/Style.css

```
@@ -0,0 +1,640 @@
+.container {
+    max-width: 810px;
+    margin: 0 auto;
+}
+@media (min-width: 1025px) {
+    body {
+        background-color: rgb(255,255,255);
+        max-width: 100%;
+        font-family:"Sourse Sans Pro";
+        box-sizing: border-box;
+    }  
+    *, *::after, *::before {
+        box-sizing: inherit;
+    }
+    header {
+       display: flex;
+       max-width: 1025px;
+       margin: auto;
+    }
+    .header__top{
+        display: flex;
+        min-width: auto;
+        gap: 0px 34px;
+    }
+    .top__menu {
+        display: block;
+        margin:33px auto auto 38px;
+        
+    }
+    .header__orangeline{
+        background-color: #C98C4D;
+        margin: auto;
+        height: 4px;
+        width: 31px;
+        align-items: center;
+        box-sizing: border-box;
+    }
+    .header__greenline {
+        background-color: #4F7F76;
+        margin: 3px auto auto auto;
```

```suggestion
        margin: 3px auto auto;
```

чекни, как тиражирование работает) https://developer.mozilla.org/ru/docs/Web/CSS/margin